flexinput/src/main/java/com/lytefast/flexinput/fragment/EmojiGridFragment.java

```diff
@@ -38,7 +35,16 @@
   private Unbinder unbinder;
 
   private Emoji.EmojiCategory emojiCategory;
+  private FlexInputFragment flexInputFrag;
 
+  @Override
+  public void onCreate(@Nullable final Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    Fragment parentFrag = getParentFragment();
```

Comment:
Perhaps use a base (private or package private) Fragment here to abstract away the boiler of setting up the flexInputFrag.

This setup code is done in all the fragments.